pkg/sync/sync.go

```diff
@@ -0,0 +1,88 @@
+package sync
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
+	sourcev1 "github.com/fluxcd/source-controller/api/v1beta1"
+)
+
+const (
+	bootstrapSourceManifest        = "toolkit-source.yaml"
+	bootstrapKustomizationManifest = "toolkit-kustomization.yaml"
+)
+
+func Generate(options Options) ([]map[string]string, error) {
```

Comment:
Should we run a struct here with the parameters? Then we can use it in the other generate function.

Reply:
Yes, let's create an `Output` struct that contains an array of `Manifest` objects, the `Manifest` could have `Path` and `Content`.